Fastlane/deployment_lanes.rb

```diff
@@ -266,6 +266,25 @@
   release(hotfix: true)
 end
 
+desc 'Create a build for running tests on browser stack'
+lane :appium_build do |options|
+  scheme = options[:scheme] || ENV['XCODE_SCHEME']
+  clear_derived_data
+  # Set timeout to prevent xcodebuild -list -project to take to much retries.
+  ENV['FASTLANE_XCODEBUILD_SETTINGS_TIMEOUT'] = '120'
+  ENV['FASTLANE_XCODE_LIST_TIMEOUT'] = '120'
+
+  gym(
+    scheme: scheme,
+    configuration: 'Debug',
+    export_method: 'development',
+    xcconfig: options[:xcconfig] || ENV['BETA_XCCONFIG'],
+    cloned_source_packages_path: 'SourcePackages'
```

Comment:
This should be `.build` now since latest changes

```suggestion
    cloned_source_packages_path: '.build'
```